web/packages/teleport/src/AuthConnectors/AuthConnectorEditor/AuthConnectorEditorContent.tsx

```diff
@@ -0,0 +1,141 @@
+import { Attempt } from 'shared/hooks/useAttemptNext';
+
+import { DesktopDescription } from 'teleport/AuthConnectors/styles/AuthConnectors.styles';
+import { FeatureBox, FeatureHeaderTitle } from 'teleport/components/Layout';
+
+import { description } from '../AuthConnectors';
+
+/**
+ * AuthConnectorEditorContent is a the content of an Auth Connector editor page.
+ */
+export function AuthConnectorEditorContent({
+  title,
+  content,
+  backButtonRoute,
+  isSaveDisabled,
+  saveAttempt,
+  fetchAttempt,
+  onSave,
+  onCancel,
+  setContent,
+}: Props) {
+  return (
+    <FeatureBox>
+      <FeatureHeaderTitle py={3} mb={2}>
+        <Flex alignItems="center">
+          <ArrowBack
+            as={RouterLink}
+            mr={2}
+            size="large"
+            color="text.main"
+            to={backButtonRoute}
+          />
+          <Flex mr={4} alignItems="baseline">
+            <H1>{title}</H1>
+          </Flex>
+        </Flex>
+      </FeatureHeaderTitle>
+      {fetchAttempt.status === 'failed' && (
+        <Alert>{fetchAttempt.statusText}</Alert>
+      )}
+      {fetchAttempt.status === 'processing' && (
+        <Box textAlign="center" m={10}>
+          <Indicator />
+        </Box>
+      )}
+      {fetchAttempt.status === 'success' && (
+        <Flex width="100%" height="100%">
+          <Flex
+            alignItems="start"
+            flexDirection={'column'}
+            height="100%"
+            flex={4}
+          >
+            {saveAttempt.status === 'failed' && (
+              <Alert>{saveAttempt.statusText}</Alert>
+            )}
+            <Flex height="600px" width="100%">
+              {content && (
+                <TextEditor
+                  bg="levels.deep"
+                  readOnly={false}
+                  data={[{ content, type: 'yaml' }]}
+                  onChange={setContent}
+                />
+              )}
+            </Flex>
+            <Box mt={3}>
+              <ButtonPrimary disabled={isSaveDisabled} onClick={onSave} mr="3">
+                Save Changes
+              </ButtonPrimary>
+              <ButtonSecondary
+                disabled={saveAttempt.status === 'processing'}
+                onClick={onCancel}
+              >
+                Cancel
+              </ButtonSecondary>
+            </Box>
+          </Flex>
+          <DesktopDescription>
+            <H3 mb={3}>Auth Connectors</H3>
+            <P mb={3}>{description}</P>
+            <P mb={2}>
+              Please{' '}
+              <Link
+                color="text.main"
+                // This URL is the OSS documentation for auth connectors
+                href="https://goteleport.com/docs/setup/admin/github-sso/"
+                target="_blank"
+              >
+                view our documentation
+              </Link>{' '}
+              on how to configure a GitHub connector.
+            </P>
+          </DesktopDescription>
```

Comment:
i think this should update to whatever type of connector we are trying to add. currently this says github even when on OIDC and SAML